Automattic/Gravatar-SDK-Android
# QuickEditor: Profile Card in the Quick Editor

Closes #254 

### Description

This PR adds the Profile Card component at the top of the Quick Editor. 

The `ProfileSummary` component from the `:gravatar-ui` module was used and put inside a Card. 
For the dark theme we can't have the shadow so the background is uses a variation of the `surface` color to be different from the background. 

| Light | Dark |
|---|---|
| <img width="692" alt="Screenshot 2024-08-06 at 12 47 42" src="https://github.com/user-attachments/assets/cc21e406-d401-484c-8ea2-eaef8a20ee48"> | <img width="692" alt="Screenshot 2024-08-06 at 12 47 50" src="https://github.com/user-attachments/assets/f277f5dc-374a-493b-b562-b5c32f95060a"> |

Note: Shadows do not work with screenshot testing so the recorded images look slightly different in a light theme. 


**Important Note**: The backend is currently returning the wrong format for the `updated date` field. As a workaround, you can apply this patch to comment that field from the Avatar object.
[Gravatar-Android-13-04-48.patch](https://github.com/user-attachments/files/16455284/Gravatar-Android-13-04-48.patch)

### Testing Steps 
1. Run the DemoApp 
2. Go to Avatar Update tab
3. Tap `Update Avatar` button
4. Follow the steps to log in 
5. Confirm you can see the profile component at the top
6. Run `adb shell cmd uimode night yes` to enable dark mode
7. Confirm the UI looks fine
